task/sast-find-unicode-control/0.1/sast-find-unicode-control.yaml

```diff
@@ -0,0 +1,232 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: sast-find-unicode-control
+spec:
+  description: >-
+    Scans source code for non-printable unicode characters in all text files.
+  results:
+    - description: Tekton task test output.
+      name: TEST_OUTPUT
+  params:
+    - description: Image URL.
+      name: image-url
+      type: string
+      # In a future 0.4 version of the task, drop the default to make this required
+      default: ""
+    - description: Image digest to report findings for.
+      name: image-digest
+      type: string
+      # In a future 0.4 version of the task, drop the default to make this required
+      default: ""
+    - name: FIND_UNICODE_CONTROL_GIT_URL
+      type: string
+      description: URL from repository to find unicode control
+      default: "https://github.com/siddhesh/find-unicode-control.git"
+    - name: KFP_GIT_URL
+      type: string
+      description: URL from repository to download known false positives files
+      default: "https://gitlab.cee.redhat.com/osh/known-false-positives.git"
+    - default: ""
+      description: Name-Version-Release (NVR) of the scanned project, used to find path exclusions (it is optional)
+      name: PROJECT_NVR
+      type: string
+    - default: ""
+      description: File to store all excluded findings to (it is optional)
+      name: RECORD_EXCLUDED
+      type: string
```

Comment:
I think this should be a boolean.  I would prefer to make all the SAST scanning CI tasks use the same file name for this to avoid special-casing in the tools/services that will consume the data from Konflux.

Reply:
yes, it should be boolean, but the konflux not support boolean, it only support string, array, so I change the description, here use `true` or `false`